Xamarin.UserConsent/IUserConsent.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Threading.Tasks;
+using Xamarin.Forms.StyleSheets;
+
+namespace Xamarin.UserConsent
+{
+    public interface IUserConsent
+    {
+        Task<ConsentFormResult> DisplayConsentRequest(DisplayConsentFormRequest request, StyleSheet popupStyles = null);
```

Comment:
I'm not convinced about having to provide styles upon each call to this method. What do you think about accepting this stylesheet in a constructor and/or a property to set a default. It still could be overridden on call-by-call basis.